exercises/practice/palindrome-products/.meta/example_2.rb

```diff
@@ -0,0 +1,52 @@
+class Palindrome
+  attr_reader :value, :factors
+  private
+  def initialize(palindrome)
+    @value, @factors = palindrome
+  end
+end
+
+class Palindromes
+
+  private
+
+  attr_reader :range, :candidates
+
+  def initialize(min_factor: 1, max_factor: 9)
+    @range = (min_factor..max_factor)
+  end
+
+  def factors(palindrome)
+    range.each_with_object([]) do |number, factors|
+      div, mod = palindrome.divmod(number)
```

Comment:
We use full names just about everywhere else, we could use `divisor` or `divisor` or whatever it is this abbreviation means and `modulo` or, is it `modulus`, here (which can make it easier for those unfamiliar with these things to find the words in their dictionaries).  Code does not have to read cryptically like code.